ros/msg/BasicNavigationFeedback.msg

```diff
@@ -0,0 +1,11 @@
+uint8 SUCCESS = 0
```

Comment:
it might be better to start at 1. If `status` is not set (for whatever reason), it will default to 0 and whoever is checking the status might think it has succeeded.

Reply:
https://github.com/ropod-project/basic_navigation/pull/3/commits/e7244f549ece4497fb7dcf889757f9102020d8c2